internal/config/config.go

```diff
@@ -160,7 +160,7 @@ func GetConfig() (*Config, error) {
 func InitConfig() {
 	viper.SetEnvPrefix("IZE")
 	viper.AutomaticEnv()
-	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
+	viper.SetEnvKeyReplacer(strings.NewReplacer("_", "-"))
```

Comment:
@psihachina Can you please give more info on why this change is required?
What tests have we done to validate that ENV, flags and configs are still working?

Reply:
@AutomationD 
I don't think it's necessary at all

Please check out these examples:
https://github.com/spf13/viper/blob/0add84cdf25b13a28c405bf03ab1acfa9b35c5db/viper_test.go#L851
https://github.com/spf13/viper/blob/0add84cdf25b13a28c405bf03ab1acfa9b35c5db/viper_test.go#L713

Reply:
Let's remove it then